### ISSUE TITLE
Kinesis event generator uses urlsafe_b64encode instead of b64encode

### ISSUE THREAD

**Issue by sonya** [awslabs/aws-sam-cli#503]
**Description:**

The code that generates Kinesis events uses `base64.urlsafe_b64encode` instead of `base64.b64encode`. This is inconsistent with earlier versions of SAM CLI. I don't know which version is correct, but there is no documentation on AWS that says they use URL-safe Base64 encoding.

https://github.com/awslabs/aws-sam-cli/blob/9dcd0bc040b0b9c61d7b61cf36748d25de5ec0e5/samcli/commands/local/generate_event/kinesis/cli.py#L45

This affects parsers downstream that use `base64.b64decode`, causing them to raise `Incorrect padding` on certain payloads.

**Steps to reproduce the issue:**
1. Generate a Kinesis event using the following sample data
```sh
sam local generate-event kinesis --data '{"user_agent": "Mozilla/5.0 (compatible; SemrushBot/2~bl; +http://www.semrush.com/bot.html)"}'
```
2. Write the above output to a file `event.json`
3. Attempt to decode the record using `base64.b64decode`
```python
import base64, json
contents = open('event.json', 'r').read()
event = json.loads(contents)
record = event['Records'][0]
base64.b64decode(record['kinesis']['data'])
```

**Observed result:**
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/base64.py", line 76, in b64decode
    raise TypeError(msg)
TypeError: Incorrect padding
```

**Expected result:**
```
'{"user_agent": "Mozilla/5.0 (compatible; SemrushBot/2~bl; +http://www.semrush.com/bot.html)"}'
```

This expected result is the result that I get when repeating the above steps with sam version 0.2.11

**Output of `sam --version`**:

SAM CLI, version 0.4.0

**Comment by jfuss** [comment]
We should double check this by running the ones that fail locally in the cloud before going further, but if we do have this wrong, it is a pretty simple update.

**Comment by jfuss** [comment]
We updated this to only [b64decode](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/commands/local/lib/generated_sample_events/events.py#L79) as part of the wider `generate-event` update.

Marking this as waiting for release.

**Comment by jfuss** [comment]
Closing this as it was released in [0.6.0](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.6.0)
